src/main/java/games/strategy/triplea/delegate/MustFightBattle.java

```diff
@@ -1821,13 +1824,13 @@ private void checkForUnitsThatCanRollLeft(final IDelegateBridge bridge, final bo
    */
   private void submergeSubsVsOnlyAir(final IDelegateBridge bridge) {
     // if All attackers are AIR submerge any defending subs ..m_defendingUnits.removeAll(m_killed);
-    if (Match.allMatch(m_attackingUnits, Matches.UnitIsAir) && Match.someMatch(m_defendingUnits, Matches.UnitIsSub)) {
+    if ( Match.allMatch(m_attackingUnits, Matches.UnitIsAir) && Match.someMatch(m_defendingUnits, Matches.UnitIsSub)) {
```

Comment:
I'm probably stating something that everyone already knows, but if you're using Eclipse, there are TripleA formatter and cleanup configurations available [here](https://github.com/triplea-game/triplea/tree/master/eclipse/format) (they can be used from IntelliJ, too, with [a little more work](https://github.com/krasa/EclipseCodeFormatter#instructions)).  I've gotten into the habit of running **Source > Clean Up...** on the files I've changed before committing so that I don't have to worry that I fat-fingered a couple of spaces (among other things like removing unused imports, making variables `final`, etc.). 😃